### ISSUE TITLE
SSE

### ISSUE THREAD

**Issue by zygimantas** [rasmus-s/stackoverflow-PushStreamContent#1]
Have you found the way to use SSE with ASPNET Core?

**Comment by rasmus-s** [comment]
No. I did never find a way to get it to work right. Please drop me a comment if you are lucky to find a solution.
